### PR TITLE
T-31 Add config sections with urls to bank sites, auto flea market, realestate site

### PR DESCRIPTION
**Pull request by goremukin**
Added config sections, tests and script fix

### CODE REVIEW

File: PortfolioKeeper/Configuration/BankLinks.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿using System.Configuration;
+
+namespace PortfolioKeeper.Configuration
+{
+    public class BankLinks : ConfigurationElement
+    {
+        [ConfigurationProperty("id", IsRequired = true, IsKey = true)]
+        public int Id
+        {
+            get { return (int)base["id"]; }
```
Comment (AntonBessarabov) [inline PortfolioKeeper/Configuration/BankLinks.cs]: Use expression body for properties, please

File: PortfolioKeeper/Configuration/BankLinks.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        }
+
+        [ConfigurationProperty("name", IsRequired = true)]
+        public string Name
+        {
+            get { return (string)base["name"]; }
```
Comment (AntonBessarabov) [inline PortfolioKeeper/Configuration/BankLinks.cs]: Use expression body for properties, please

File: PortfolioKeeper/Configuration/BankLinks.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        }
+
+        [ConfigurationProperty("exchangeRatesLink", IsRequired = true)]
+        public string ExchangeRatesLink
+        {
+            get { return (string)base["exchangeRatesLink"]; }
```
Comment (AntonBessarabov) [inline PortfolioKeeper/Configuration/BankLinks.cs]: Use expression body for properties, please

File: PortfolioKeeper/Configuration/BankLinks.cs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        }
+
+        [ConfigurationProperty("ingotsLink", IsRequired = true)]
+        public string IngotsLink
+        {
+            get { return (string)base["ingotsLink"]; }
```
Comment (AntonBessarabov) [inline PortfolioKeeper/Configuration/BankLinks.cs]: Use expression body for properties, please

File: PortfolioKeeper/Configuration/BankLinksCollection.cs [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+{
+    public class BankLinksCollection : ConfigurationElementCollection
+    {
+        internal const string PropertyName = "bank";
+
+        public override ConfigurationElementCollectionType CollectionType
+        {
+            get
+            {
+                return ConfigurationElementCollectionType.BasicMapAlternate;
```
Comment (AntonBessarabov) [inline PortfolioKeeper/Configuration/BankLinksCollection.cs]: Use expression body for properties, please

File: PortfolioKeeper/Configuration/BankLinksCollection.cs [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+            }
+        }
+        protected override string ElementName
+        {
+            get
+            {
+                return PropertyName;
```
Comment (AntonBessarabov) [inline PortfolioKeeper/Configuration/BankLinksCollection.cs]: Use expression body for properties, please

File: PortfolioKeeper/Configuration/BankLinksCollection.cs [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        }
+
+        protected override object GetElementKey(ConfigurationElement element)
+        {
+            return ((BankLinks)(element)).Id;
+        }
+
+        public BankLinks this[int idx]
+        {
+            get { return (BankLinks)BaseGet(idx); }
```
Comment (AntonBessarabov) [inline PortfolioKeeper/Configuration/BankLinksCollection.cs]: Use expression body for properties, please

File: PortfolioKeeper/Configuration/OnlinerSection.cs [hunk 8]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Configuration;
+
+namespace PortfolioKeeper.Configuration
+{
+    public class OnlinerSection : ConfigurationSection
+    {
+        [ConfigurationProperty("carsLinks")]
+        public CarsLinks CarsLinks => (CarsLinks)this["carsLinks"];
```
Comment (AntonBessarabov) [inline PortfolioKeeper/Configuration/OnlinerSection.cs]: I think it's better to call it `OnlinerCarLinks` or smth like this

File: PortfolioKeeper/Configuration/RealtbySection.cs [hunk 9]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Configuration;
+
+namespace PortfolioKeeper.Configuration
+{
+    public class RealtbySection : ConfigurationSection
+    {
+        [ConfigurationProperty("apartmentsLinks")]
+        public ApartmentsLinks ApartmentsLinks => (ApartmentsLinks)this["apartmentsLinks"];
```
Comment (AntonBessarabov) [inline PortfolioKeeper/Configuration/RealtbySection.cs]: I think it's better to call it `RealtbyApartmentLinks` or smth like this

File: PortfolioKeeper/Configuration/BankLinks.cs [hunk 10]
```diff
@@ -0,0 +1,31 @@
+﻿using System.Configuration;
+
+namespace PortfolioKeeper.Configuration
+{
+    public class BankLinks : ConfigurationElement
+    {
+        [ConfigurationProperty("id", IsRequired = true, IsKey = true)]
+        public int Id
```
Comment (EdwOK) [inline PortfolioKeeper/Configuration/BankLinks.cs]: `IntegerValidator(..)`

File: PortfolioKeeper/Configuration/BankLinks.cs (hunk 2 above)
Comment (EdwOK) [reply PortfolioKeeper/Configuration/BankLinks.cs]: `StringValidator`

File: PortfolioKeeper/Configuration/BankLinks.cs (hunk 3 above)
Comment (EdwOK) [reply PortfolioKeeper/Configuration/BankLinks.cs]: `StringValidator`

File: PortfolioKeeper/Configuration/BankLinks.cs (hunk 4 above)
Comment (EdwOK) [reply PortfolioKeeper/Configuration/BankLinks.cs]: `StringValidator`

File: PortfolioKeeper/Configuration/CarsLinks.cs [hunk 11]
```diff
@@ -0,0 +1,28 @@
+﻿using System.Configuration;
+
+namespace PortfolioKeeper.Configuration
+{
+    public class CarsLinks : ConfigurationElement
```
Comment (EdwOK) [inline PortfolioKeeper/Configuration/CarsLinks.cs]: Validation attributes

**Pull request merged**